### PR TITLE
Drop the custom git sha for ratbagd.devel

### PR DESCRIPTION
**Pull request by whot**
After spending way too much time working around the git sha instead of getting useful stuff done, I've decided to drop it. Our `.devel` tools are supposed to be for developers but right now they get in the way more than anything - try to debug why ratbagd crashes when you're running a ratbagctl.test, it's insane.

Anyway, first bunch of patches replaces the `org.freedesktop.ratbag_devel1_$gitsha` with just `org.freedesktop.ratbag_devel1` as busname. Nothing too exciting here. Second-to-last patch moves installing the dbus file from the tool box to `ratbagd.devel` directly, this way it can just be started as-is.

Last patch is just a cosmetic fix unrelated, because we can't be pretty enough.

### CODE REVIEW

**Comment by FFY00** [comment]
> try to debug why ratbagd crashes when you're running a ratbagctl.test, it's insane

I just run `ratbagd` directly.

File: meson.build [hunk 1]
```diff
@@ -409,21 +417,35 @@ executable('ratbagd',
 install_man('ratbagd/ratbagd.8')
 
 #### ratbagd_devel ####
+#
+# A development ratbagd server that owns a different name on the bus (named
+# after the git sha). Do not use directly, this server is used by
```
Comment (FFY00) [inline meson.build]: Not anymore :laughing:.

**Comment by FFY00** [comment]
> try to debug why ratbagd crashes when you're running a ratbagctl.test, it's insane

I just run `ratbagd` directly.

**Comment by whot** [comment]
`ratbagd` doesn't use the data files from the source repo but loads it from the prefix. And it doesn't initialize the test device, which is needed when you're testing random things. Those two are the main motivation for `ratbagctl.devel`.

File: meson.build (hunk 1 above)
Comment (whot) [reply meson.build]: fixed! thanks for the review. I also squashed in the removal of the git-sha-generating script.

**Pull request merged**